plugin/teksi_wastewater/interlis/interlis_importer_exporter.py

```diff
@@ -97,8 +97,8 @@ def interlis_import(self, xtf_file_input, show_selection_dialog=False, logs_next
         self._import_xtf_file(xtf_file_input=xtf_file_input)
 
         # Disable symbology triggers
-        self._progress_done(35, "Disable symbolgy triggers...")
-        self._import_disable_symbology_triggers()
+        self._progress_done(35, "Disable symbology and modification triggers...")
+        self._import_disable_symbology_and_modification_triggers()
```

Comment:
If you deactivate your modification triggers, you won't be able to tell what were the entities modified by your import. I would suggest to deactivate only symbology triggers.

Reply:
not possible: In AG-64/96, the inserts/updates are handled by triggers, not deactivating the modification triggers causes psycopg problems due to pending trigger events.
Also, we need to disable the last_modification triggers because otherwise we won't be able to import the base data.

Reply:
Ok then let's keep it that way for now and tackle it in the import/export issue 👍